java/core/src/main/java/sleeper/core/table/TableNotFoundException.java

```diff
@@ -33,11 +33,11 @@ public static TableNotFoundException withTableName(String tableName) {
         return new TableNotFoundException("Table not found with name \"" + tableName + "\"");
     }
 
-    public static TableNotFoundException withTableIdentity(TableIdentity tableIdentity) {
-        return withTableIdentity(tableIdentity, null);
+    public static TableNotFoundException withTableIdentity(TableStatus table) {
+        return withTableIdentity(table, null);
     }
 
-    public static TableNotFoundException withTableIdentity(TableIdentity tableIdentity, Exception cause) {
-        return new TableNotFoundException("Table not found " + tableIdentity, cause);
+    public static TableNotFoundException withTableIdentity(TableStatus table, Exception cause) {
```

Comment:
These methods can be renamed to `withTableStatus`

Reply:
Done, thanks!